tools/build_langserver/lsp/test_data/src/core/test.build

```diff
@@ -6,6 +6,7 @@ go_library(
             "*_test.go",
             "version.go",
         ],
+        allow_empty = True,
```

Comment:
lps crashes if this is not included, see https://github.com/thought-machine/please/issues/2772#issuecomment-1520206577